core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -454,6 +468,8 @@ angular.module('oppia').directive('storyNodeEditor', [
               )
             );
             _init();
+
+            $scope.getPrerequisiteSkillsDescription();
```

Comment:
It seems like `skillIdToSummaryMap` is initialised inside `_init`, so it seems odd to initialise it again afterwards. Can we initialise it the correct way the first time?

Reply:
I have called the getPrerequisiteSkillsDescription function inside of _init instead of calling it separably in onInit hook.